diff --git a/node_modules/@mui/x-license-pro/verifyLicense/verifyLicense.js b/node_modules/@mui/x-license-pro/verifyLicense/verifyLicense.js
index 259519d..e7b5b8a 100644
--- a/node_modules/@mui/x-license-pro/verifyLicense/verifyLicense.js
+++ b/node_modules/@mui/x-license-pro/verifyLicense/verifyLicense.js
@@ -92,7 +92,9 @@ export function verifyLicense({
   acceptedScopes,
   isProduction
 }) {
-  if (!releaseInfo) {
+  console.log('Do what you want cause a pirate is free. You are a pirate');
+
+  /*if (!releaseInfo) {
     throw new Error('MUI: The release information is missing. Not able to validate license.');
   }
 
@@ -147,7 +149,7 @@ export function verifyLicense({
 
   if (!acceptedScopes.includes(license.scope)) {
     return LicenseStatus.Invalid;
-  }
+  }*/
 
   return LicenseStatus.Valid;
 }
\ No newline at end of file
